deploy/prod/chainlog-logger.yaml

```
@@ -0,0 +1,60 @@
+replicaCount: 1
+service:
+  name: chainlog-logger
+  port: 8080
+  type: ClusterIP
+  containerPort: 8080
+  tls:
+    enabled: false
+image:
+  repository: ${ECR_REGISTRY}/chainlog-logger-prod
+  pullPolicy: IfNotPresent
```

Isn't it best to use `Always` to avoid the possibility of using an old image whenever you need the latest one? It won't download the image if it matches the digest of the locally stored image, as per [K8S docs](https://kubernetes.io/docs/concepts/containers/images/#:~:text=Always,launch%20the%20container.), so new pod deployments won't be unnecessarily slowed down.

Leaving the suggestion in case you want to merge it:

```suggestion
  pullPolicy: Always
```